fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/QueryPlanResult.java

```diff
@@ -39,7 +39,12 @@
 
     public QueryPlanResult(@Nonnull final RecordQueryPlan plan) {
         this.plan = plan;
-        planInfo = new QueryPlanInfo();
+        planInfo = QueryPlanInfo.EMPTY;
```

Comment:
Usually, it's better to have a static method `empty()` on `QueryPlanInfo` to return `EMPTY` when called. This allows you to encapsulate whether `QueryPlanInfo` really should just return the same instance over and over or whether you create a different instance every time. It's a pattern I follow but that's just nitpicking.

Reply:
That makes sense. Done.